### PR TITLE
Update setup.py to read requirements.txt files

### LINKED ISSUE #68: Configure and implement Pytest
This issue aims to configure a unit tests tool.

Tasks:

- [x] Define a test tool (probably pytest)
- [x] Configure it locally
- [x] Add tests to CI
- [x] Add badge of project coverage
- [x] Create the first test of the project

Is important too:

- [x] Define how to make **integration** tests (very important to `components` usage). Moved to #76 
- [x] ~~Define how to make **stress** tests~~
- [x] Define how to make **load** tests. Moved to #77

### PR DESCRIPTION
**Pull request by arthurTemporim**
Closes #68 

See full description and discussion on issue #68 .

### CODE REVIEW

**Comment by arthurTemporim** [comment]
On this PR a simple update into documentation was added too with tests content.

**Review by ralphg6 — APPROVED** [review]

File: david/README.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+# David
```
Comment (ralphg6) [inline david/README.md]: Why you duplicate README.md?

**Review by ralphg6 — CHANGES_REQUESTED** [review]
Why you duplicate README.md?

File: david/README.md (hunk 1 above)
Comment (arthurTemporim) [reply david/README.md]: It is not needed, I removed it.

**Review by ralphg6 — APPROVED** [review]

**Pull request merged**